openmoc/docstring.i

```diff
@@ -2966,70 +2911,51 @@ the FSR scalar flux
 %feature("docstring") GPUSolver::computeFSRFissionSources "
 computeFSRFissionSources()  
 
-Computes the fission source in each FSR.  
-
-This method computes the fission source in each FSR based on this iteration's current
-approximation to the scalar flux.  
+Computes the total fission source for each FSR and energy group.  
 ";
 
 %feature("docstring") GPUSolver::computeKeff "
 computeKeff()  
 
-Compute $ k_{eff} $ from successive fission sources.  
-
-This method computes the current approximation to the multiplication factor on this
-iteration as follows: $ k_{eff} = \\frac{\\displaystyle\\sum_{i \\in I}
-\\displaystyle\\sum_{g \\in G} \\nu \\Sigma^F_g \\Phi V_{i}} {\\displaystyle\\sum_{i \\in
-I} \\displaystyle\\sum_{g \\in G} (\\Sigma^T_g \\Phi V_{i} - \\Sigma^S_g \\Phi V_{i} -
-L_{i,g})} $  
+Compute $ k_{eff} $ from total fission and absorption rates in each FSR and energy group.  
```

Comment:
Actually this comment is wrong - we compute keff from successive fission sources. I'm not sure how this changed in this PR since I never touched it, but anyway.